jdaviz/configs/default/plugins/model_fitting/model_fitting.py

```diff
@@ -32,18 +33,13 @@ def __init__(self, value, unit=None):
 
 
 @tray_registry('g-model-fitting', label="Model Fitting")
-class ModelFitting(PluginTemplateMixin, SpectralSubsetSelectMixin):
+class ModelFitting(PluginTemplateMixin, DatasetSelectMixin, SpectralSubsetSelectMixin):
     dialog = Bool(False).tag(sync=True)
     template_file = __file__, "model_fitting.vue"
-    dc_items = List([]).tag(sync=True)
-    form_valid_data_selection = Bool(False).tag(sync=True)
     form_valid_model_component = Bool(False).tag(sync=True)
 
-    selected_data = Unicode("").tag(sync=True)
```

Comment:
There are still lines in this file (428 to 431) that use `self.selected_data` and that caused an error for me when doing model fitting.

Reply:
woah, good catch!  I swear I tested cube fitting... no idea why that didn't show up.  I now did a fully text-search and there are no other remaining `self.selected_data` instances.